src/scenes/Wallet/styles.js

```diff
@@ -0,0 +1,14 @@
+const styles = (theme) => ({
+  infoPaper: {
+    minWidth: theme.sizes.table.minWidth,
+    overflowX: 'scroll',
+    boxShadow: 'none',
+    borderRadius: theme.borderRadius,
+    background: theme.palette.background.paper,
+    paddingTop: '10px',
+    paddingLeft: theme.padding.md.px,
```

Comment:
when setting padding, its best to make it symmetrical. so top/bottom and left/right should be the same. you can do it like this:

```
padding: `${theme.padding.xs.px} ${theme.padding.md.px}`
```

The first value is the top/bottom, and the second value is left/right.